Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-03-code-0003-previous.swift

```diff
@@ -39,10 +39,9 @@ struct SyncUpDetail {
     Reduce { state, action in
       switch action {
       case .alert(.presented(.confirmButtonTapped)):
-        return .run { send in
-          await send(.delegate(.deleteSyncUp(id: state.syncUp.id)))
-          await dismiss()
-        }
+        @Shared(.fileStorage(.syncUps)) var syncUps: IdentifiedArrayOf<SyncUp> = []
+        syncUps.remove(id: state.syncUp.id)
+        return .run { _ in await dismiss() }
```

Comment:
This was brought up in an email, but we had some old delegate code in the tutorial still.